src/System.Net.Http.WinHttpHandler/tests/UnitTests/WinHttpHandlerTest.cs

```diff
@@ -846,6 +846,20 @@ public void SendAsync_NonSecureRequestToServer_ExpectNoClientCertContext()
                 async () => { await client.SendAsync(request, HttpCompletionOption.ResponseHeadersRead, cts.Token); });
         }
 
+        [Fact]
+        public async Task SendAsync_WinHttpOpenReturnsError_ExpectHttpRequestException()
+        {
+            var handler = new WinHttpHandler();
+            var client = new HttpClient(handler);
+            var request = new HttpRequestMessage(HttpMethod.Get, FakeServerEndpoint);
+
+            TestControl.Fail.WinHttpOpen = true;
+
+            Exception ex = await Assert.ThrowsAsync<HttpRequestException>(
+                async () => { await client.SendAsync(request); });
```

Comment:
Ah, of course, that makes sense.  But you should still be able to do:
Exception ex = await Assert.ThrowsAsync<HttpRequestException>(() => client.SendAsync(request));
Again, my primary feedback was about the unnecessary async/await.  The above compiles, no?